### PR TITLE
docs: rename jasmine_test file

### LINKED ISSUE #56: [Bug]: macro is not named like the rule
### What happened?

```
$ bazel query --output=label_kind examples/...
_copy_file rule //examples/commonjs:_test_copy_jasmine_junit_reporter
_copy_file rule //examples/commonjs:_test_copy_jasmine_runner
_jasmine_test rule //examples/commonjs:test
_copy_file rule //examples/simple:_test_copy_jasmine_junit_reporter
_copy_file rule //examples/simple:_test_copy_jasmine_runner
_jasmine_test rule //examples/simple:test
```

### Version

Latest

### How to reproduce

_No response_

### Any other information?

_No response_

### PR DESCRIPTION
**Pull request by alexeagle**
This ensures that e.g. --test_lang_filters=jasmine will work as expected

Fixes #56

BREAKING CHANGE:
- generate_junit_xml is now named jasmine_reporters

### CODE REVIEW

File: jasmine/defs.bzl [hunk 1]
```diff
@@ -1,16 +1,9 @@
 "Public API re-exports"
 
 load("@aspect_bazel_lib//lib:copy_file.bzl", "copy_file")
-load("@aspect_rules_jasmine//jasmine/private:jasmine_test.bzl", "lib")
-load("@aspect_rules_js//js:libs.bzl", "js_binary_lib")
+load("@aspect_rules_jasmine//jasmine/private:jasmine_test.bzl", _jasmine_test = "jasmine_test")
 
-_jasmine_test = rule(
```
Comment (jbedard) [inline jasmine/defs.bzl:7]: Was there a docs reason for moving this?

File: docs/BUILD.bazel [hunk 2]
```diff
@@ -3,7 +3,7 @@
 load("@aspect_bazel_lib//lib:docs.bzl", "stardoc_with_diff_test", "update_docs")
 
 stardoc_with_diff_test(
-    name = "rules",
+    name = "jasmine_test",
     bzl_library_target = "//jasmine:defs",
```
Comment (jbedard) [inline docs/BUILD.bazel:7]: Labelling all "defs" as "jasmine_test" seems odd but I guess we can clean that up if we ever add more rules 🤷 

Same thing with jest and I'm sure others that have a simple enough defs.bzl that everything is in it.

File: jasmine/defs.bzl (hunk 1 above)
Comment (alexeagle) [reply jasmine/defs.bzl:7]: It fixes #56 - no that could possibly be in a separate PR from the docs rename.

File: docs/BUILD.bazel (hunk 2 above)
Comment (alexeagle) [reply docs/BUILD.bazel:7]: we can document a subset of the symbols in `defs.bzl` like https://github.com/bazel-contrib/rules_oci/blob/main/docs/BUILD.bazel#L5-L12

File: README.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
 # Bazel rules for Jasmine
 
+Runs the [Jasmine](https://jasmine.github.io/) JS testing tool under Bazel.
```
Comment (jbedard) [inline README.md:3]: Do we want the "rules_* is just a part of what Aspect provides: ..." that we have in others such as jest?

**Review by jbedard — APPROVED** [review]

File: docs/BUILD.bazel (hunk 2 above)
Comment (alexeagle) [reply docs/BUILD.bazel:7]: I changed this to only document `jasmine_test` and not the underlying `jasmine_test_rule`

**Pull request merged**